sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedResponse.java

```diff
@@ -21,6 +24,9 @@
      * @return The items in the page.
      */
     default List<T> getValue() {
```

Comment:
Should this be marked as deprecated as it is in Page?

Reply:
`PagedResponse<T>` is both `Page<T>` and `Response<List<T>>`. When user consume it as `Response<List<T>>` type, say passing `PagedResponse<T>` instance to a method that takes `Response<List<T>>`, the `getItems()/getElements()` are not available in this context, so in this context if we deprecate `getValue()` then there is no alternate way to access the content.

Reply:
So my concern is - Is it legit to mark  `PagedResponse<T>::getValue()` "deprecated" when the contract `Response<T>::getValue()` it implements is "not deprecated". @JonathanGiles thoughts?

Reply:
Anu's final point is the right perspective on this. We can't say that `getValue` is deprecated in `PagedResponse` without first deprecating `getValue` in Response. Before this PR is merged though we should very carefully review all deprecations to ensure that they are correct and appropriate.

Reply:
thanks Jonathan for the clarification, learned something new about deprecation.